loops.js

```diff
@@ -50,4 +64,12 @@
 // >> 8
 // >> 9
 // ==========================================
-
+for (let i = 0; i < 9; i++) {
```

Comment:
Doordat je i > 9 hebt staan gaat je output tot 8 en niet zoals in de opdracht tot en met 9.